### PR TITLE
User can now navigate back to home page

### PR DESCRIPTION
**Pull request by satyamgupta1495**
Users can now navigate back to the home page and also fixed some minor alignment issues.

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -1,33 +1,57 @@
...
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
```
Comment (Abirpal202049) [inline contact.html]: Change the title of the page

File: open.css [hunk 2]
```diff
@@ -1,66 +1,73 @@
...
+  width: 50%;
+  text-align: center;
 }
 
 body {
-    background: url("contactbg.png");
-    background-size: cover;
-    background-repeat: no-repeat;
+  background: url("contactbg.png");
+  background-size: cover;
```
Comment (Abirpal202049) [inline open.css:25]: Targed the body by class or id

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]

**Review by thepranaygupta — APPROVED** [review]
just update the page title

**Comment by satyamgupta1495** [comment]
is there any more changes required?

**Comment by shivshankar9** [comment]
Resolve conflicts @satyamgupta1495

**Comment by satyamgupta1495** [comment]
> Resolve conflicts @satyamgupta1495

Don't have the write access. Can you tell me how can I resolve it?

**Review by Abirpal202049 — APPROVED** [review]

**Pull request merged**